config.yaml

```
@@ -123,7 +125,20 @@ groot:
     maxlength: 125           # Maxlength for groot index
     covcutoff: 0.97          # Coverage cutoff for groot report
     lowcov: False            # Report ARGs with no 5' or 3' coverage. Overrides covcutoff.
-
+amrplusplus:
+    databases:
+        db: "scripts/amrplusplus/db/megares_modified_database_v2.00.fasta"
+        annotation: "scripts/amrplusplus/db/megares_modified_annotations_v2.00.csv"
```

```suggestion
        megares_annotation: "scripts/amrplusplus/db/megares_modified_annotations_v2.00.csv"
```
To make it more clear when reading references to this in rules

---

Don't forget to update the references in the code as well ;)